new-frontiers-14/frontier-station-14
# Pumpkin Spice Latte

## About the PR



Even in space, autumn is coming.  Adds a craftable pumpkin spice latte, along with metamorphic glass sprites by Stagnation (many thanks).

## Why / Balance


Enhancing barista roleplay (combine with #1546 for maximum effect), gives ships more incentive to grow pumpkins.  Adds _NF files for flavours/drinks that can help springboard newer devs into making more food recipes (#1159).

## How to test


1. Run locally.
2. Make some pumpkin spice lattes.
3. Observe the glass sprites as you drink the sickly sweet nectar.
4. Appreciate the fine orange colour of the liquid in a jug.

## Media


![pumpkin-spice-slurp](https://github.com/new-frontiers-14/frontier-station-14/assets/166147148/99c6a782-9379-47a9-b32c-3f595df78689)

![pumpkin-spice-guidebook](https://github.com/new-frontiers-14/frontier-station-14/assets/166147148/66d80237-3980-4c97-8968-a83dfb3728c7)


## Breaking changes


**Changelog**



:cl: whatston3 and Stagnation
- add: Spiced pumpkin lattes can now be mixed by bartenders.
